### PR TITLE
Kube burner ocp

### PR DESCRIPTION
**Pull request by paigerube14**
Updating kube burner ocp to use the e2e-benchmarking version and stay consistent with our setup

### CODE REVIEW

**Comment by paigerube14** [comment]
Passing run: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/e2e-benchmarking-multibranch-pipeline/job/kube-burner-ocp/6/console

File: Jenkinsfile [hunk 1]
```diff
@@ -247,18 +258,13 @@ pipeline {
...
-
+                        cd workloads/kube-burner-ocp-wrapper
                         if [[ $CHURN == true ]]; then
                             echo "churn true"
-                            churn_val="--churn=true --churn-delay=${CHURN_DELAY} --churn-duration=${CHURN_DURATION} --churn-percent=${CHURN_PERCENT}"
+                            EXTRA_FLAGS="--churn=true --churn-delay=${CHURN_DELAY} --churn-duration=${CHURN_DURATION} --churn-percent=${CHURN_PERCENT}"
                         fi
-                        pwd
-                        ${KUBE_DIR}/kube-burner ocp $WORKLOAD --iterations=$VARIABLE --timeout=6h --es-server=$ES_SERVER --es-index=ripsaw-kube-burner $churn_val |& tee "kube-burner.out"
+                        ./run.sh $churn_val |& tee "kube-burner.out"
```
Comment (qiliRedHat) [inline Jenkinsfile:267]: @paigerube14 I don't see the run.sh get $1, just set EXTRA_FLAGS for chrun settings should be enough?
https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/kube-burner-ocp-wrapper/run.sh

Comment (paigerube14) [reply Jenkinsfile]: Missed that variable, took out now

**Pull request merged**